Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.py

```diff
@@ -321,8 +324,73 @@ def parse_reports_relationships(reports: List, sub_reports: List, matched_relati
                     'value': external_id,
                     'description': ','.join(all_urls)
                 }])
+                indicator_val = external_id
+
+            # create MITRE ATT&CK indicator
+            if indicator_val and type_name == 'MITRE ATT&CK':
+
+                # populate mitre course of action data from the relevant relationships
+                relationship = relation_object.get('id')
+                courses_of_action: Dict[str, List] = {}
+
+                if matched_relationships.get(relationship):
+                    for source in matched_relationships[relationship]:
+                        if source.startswith('course-of-action') and id_to_object.get(source):
+                            relationship_product = courses_of_action_products[source]
+                            if not courses_of_action.get(relationship_product):
+                                courses_of_action[relationship_product] = []
+                            courses_of_action[relationship_product].append(id_to_object[source])
+
+                indicators.append({
+                    "value": indicator_val,
+                    "type": 'MITRE ATT&CK',
+                    "fields": {
+                        "firstseenbysource": relation_object.get('created'),
+                        "indicatoridentification": relation_object.get('id'),
+                        "tags": [],
+                        "modified": relation_object.get('modified'),
+                        "reportedby": 'Unit42',
+                        "mitrecourseofaction": create_course_of_action_field(courses_of_action)
+                    }
+                })
 
-    return reports
+    return reports, indicators
+
+
+def create_course_of_action_field(courses_of_action: dict) -> str:
+    """creates a markdown tables from the courses of action data according to the product type.
+
+    Args:
+        courses_of_action: dictionary containing the courses of action data.
+
+    Returns:
+        markdown string with courses of action tables.
+    """
+    if not courses_of_action:
+        return 'No courses of action found.'
+    markdown = ''
+    for relationship_product, courses_list in courses_of_action.items():
+        tmp_table = []
+        for course_of_action in courses_list:
+            row = {}
+            if relationship_product in ('Cortex XDR Prevent', 'DNS Security', 'XSOAR'):
```

Comment:
Create a constant for this list, it would be easier to modify it afterward if needed